interchain-exchange/tutorial/00-tutorial.md

```diff
@@ -388,6 +410,51 @@ balances:
 
 The counterparty (sender of the buy mcx order) received 5 mcx vouchers. vcx balance hasn't changed, because the correct amount of vcx (50) were locked at the creation of the buy order during the previous step.
 
+```yml
+# Target blockchain
+balances:
+- amount: "5" # increased from 0
+  denom: ibc/99678A10AF684E33E88959727F2455AE42CCC64CD76ECFA9691E1B5A32342D33 # mcx voucher
+```
+-->
+
+We now have two orders open for `marscoin`: a *sell order* on the **source chain** (for 20 `marscoin` at 5 `venuscoin`) and a *buy order* on the **target chain** (for 5 `marscoin` at 5 `venuscoin`). Let's perform an exchange by sending a sell order to the source chain.
```

Comment:
If we switch to `marscoin`, we need to change the denom in config as well as in examples.

Reply:
We can still use mcx and vcx while writing code. But for explanation I propose on using `marscoin` and `venuscoin` respectively.

Reply:
Arguably, this can confuse people, because we use `code` syntax, so either 1. `marscoin` everywhere, or 2. `mcx` everywhere, 3. or `mcx` and Mars coin.

Reply:
I vote for same in code and text, and yes, using the tick marks for code syntax for literals